clouddriver-ecs/clouddriver-ecs.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
   integrationImplementation "org.springframework:spring-test"
   integrationImplementation "org.testcontainers:mysql"
   integrationImplementation "mysql:mysql-connector-java"
+  integrationImplementation sourceSets.test.output
+  integrationImplementation sourceSets.main.output
```

Comment:
were these outputs always unnecessary (possible!), or did some here change make this so?

Reply:
I had to pull in classes from test and main modules for mocking purposes. Seems like they were not needed before. I could not get tests to run without these changes (getting `UnsatisfiedDependencyException` without). I am new to Spring so it's possible I am doing something wrong here.